infrastructure/ansible/roles/system/admin_user/tasks/main.yml

```diff
@@ -11,7 +11,7 @@
     shell: /bin/bash
     create_home: yes
     append: yes
-    groups: admin
+    groups: admin,adm
```

Comment:
adm?

Reply:
> Of some note, in this update we add 'admin' user to the 'adm' group on
application servers to allow 'admin' user sudo-less journalctl. Without
that you need to run 'sudo' to be able to view logs for arbitrary systemctl
services.

Reply:
Ah, missed that part. I was searching for this in the rest of the diff but couldn't find it.
Thanks for clarifying